doc/nobol/src/nobol7.lalrpop

```diff
@@ -0,0 +1,10 @@
+use super::{Var, Lit, Stmt};
+
+grammar;
+
+pub Var: Var = <r"[a-z]"> => <>.chars().next().unwrap().into();
+
+pub Lit: Lit = <l:r#""(\\\\|\\"|[^"])*""#> =>
+    l[1..l.len()-1].replace("\\\"", "\"").replace("\\\\", "\\").into();
```

Comment:
This would consider `\\"` as an escaped `"` rather than an as escaped `\`.

`\\"` wouldn't be accepted by the regex to begin with, but the problem is more apparent if more escape sequences were added. For example if `\n` was also supported, `\\n` would be parsed as `\<LF>`.

Maybe this could instead suggest an approach similar to that implemented in [`tok::apply_string_escapes()`](https://github.com/lalrpop/lalrpop/blob/d45124db36919e8a70a04581745b156fdeafbe92/lalrpop/src/tok/mod.rs#L749)?

Reply:
Excellent point! You're right, I was handling this in a quick and dirty fashion, and while it "works" for this simple case (to `\\"` being illegal, as you noted), it does not scale well.

Reply:
I'm using this `"([^"\\\r\n]|\\[^\r\n])*"`
It accept something like `\n`, `\t`